azext_edge/tests/edge/init/int/helper.py

```diff
@@ -0,0 +1,132 @@
+# coding=utf-8
+# ----------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License file in the project root for license information.
+# ----------------------------------------------------------------------------------------------
+
+from enum import Enum
+from typing import Any, Dict, List, Optional, Union
+from azext_edge.edge.providers.orchestration.common import KEYVAULT_ARC_EXTENSION_VERSION
+from ....helpers import run
+
+
+class ResourceKeys(Enum):
+    custom_location = "Microsoft.ExtendedLocation/customLocations"
+    dataprocessor = "Microsoft.IoTOperationsDataProcessor"
+    mq = "Microsoft.IoTOperationsMQ/mq"
+    orchestrator = "Microsoft.IoTOperationsOrchestrator/targets"
+    connected_cluster = "Microsoft.Kubernetes/connectedClusters"
+
+
+def assert_init_result(
+    result: Dict[str, Any],
+    cluster_name: str,
+    key_vault: str,
+    resource_group: str,
+    arg_dict: Dict[str, Union[str, bool]],
+    sp_app_id: Optional[str] = None,
+    sp_object_id: Optional[str] = None,
+):
+    assert result
+    assert result["deploymentState"]["status"] == "Succeeded", "AIO deployment failed"
+
+    assert result["clusterName"] == cluster_name
+    assert result["clusterNamespace"] == arg_dict.get("cluster_namespace", "azure-iot-operations")
+    assert result["deploymentLink"]
+    assert result["deploymentName"]
+    assert result["deploymentLink"].endswith(result["deploymentName"])
+
+    # CSI driver
+    assert result["csiDriver"]["keyVaultId"] == key_vault
+    assert result["csiDriver"]["kvSpcSecretName"] == arg_dict.get("kv_spc_secret_name", "azure-iot-operations")
+    assert result["csiDriver"]["version"] == arg_dict.get("csi_ver", KEYVAULT_ARC_EXTENSION_VERSION)
+
+    if sp_app_id:
+        assert result["csiDriver"]["spAppId"] == sp_app_id
+    if sp_object_id:
+        assert result["csiDriver"]["spObjectId"] == sp_object_id
+    csi_config = result["csiDriver"]["configurationSettings"]
+    enabled_rotation = 'false' if arg_dict.get("disable_rotation") else 'true'
```

Comment:
What about `--disable-rotation false`?

Reply:
was going to leave that part alone (since this is a double negative and it is the same as just not having the arg)
but I guess it would help to have it

Reply:
![image](https://github.com/Azure/azure-iot-ops-cli-extension/assets/73560279/177d3ab7-10a2-460e-90f7-017aa723c24e)